src/gui/folderwatcher.cpp

```diff
@@ -57,7 +58,7 @@ bool FolderWatcher::pathIsIgnored( const QString& path )
 
 #ifndef OWNCLOUD_TEST
     if (_folder->isFileExcludedAbsolute(path)) {
-        qDebug() << "* Ignoring file" << path;
+        qCDebug(lcFolderWatcher) << "* Ignoring file" << path;
```

Comment:
I more meant from a user perspective "oh my file is not synced, why?". But maybe that happens too seldomly..